addons/interaction/functions/fnc_laptop_close.sqf

```diff
@@ -10,6 +10,8 @@
 
 params['_laptop'];
 
+_laptop enableSimulationGlobal false;
```

Comment:
I reviewed my recommondation and this does not work, because the simulation true state
is needed for the animation to work. So we can remove this.